test/Sdk.Generator.Tests/FunctionExecutor/FunctionExecutorGeneratorTests.cs

```diff
@@ -115,11 +115,11 @@ namespace TestProject
     internal class DirectFunctionExecutor : IFunctionExecutor
     {{
         private readonly IFunctionActivator _functionActivator;
-        private readonly Dictionary<string, Type> types = new()
+        private readonly Dictionary<string, Type> types = new Dictionary<string, Type>()
```

Comment:
Approving, but as discussed, can we please update the source generator verifier used by this test to set the correct language version to make sure we don't run into similar issues in the future?